jmeter/routing.jmx

```diff
@@ -525,6 +523,15 @@
         <hashTree/>
         <RecordingController guiclass="RecordController" testclass="RecordingController" testname="Recording Controller" enabled="true"/>
         <hashTree>
+          <CounterConfig guiclass="CounterConfigGui" testclass="CounterConfig" testname="Compteur" enabled="true">
+            <stringProp name="CounterConfig.start">0</stringProp>
+            <stringProp name="CounterConfig.end">99</stringProp>
```

Comment:
can you convert this parameter in variable for command.  like this for instance : ${__P(nbr_node,99)}